### ISSUE TITLE
Sub domain Request for Production environment

### ISSUE THREAD

**Issue by romer8** [CIROH-UA/ciroh-ua.github.io#4]
**1.	Requester Information:**

Giovanni Romero
email: gromero@aquaveo.com
slack: Gio

**2.	Project Information:** 
The following information will be needed to add the subdomain of the Tethys Platform portal.:

1. Recored Type: A
2. TTL: 1 Hour
3. Data  3.221.231.152 and  44.205.207.244

**3.	name of subdomain:** 
Request of the folllowing subdomain: [portal.ciroh.org](http://portal.ciroh.org/)

**4.	Url of serving address:**
The urls that the subdomain will be mapping to the following Network Load Balancer DNS:
cirohportal-prod-nlb-e606439f7706f620.elb.us-east-1.amazonaws.com

**Comment by benlee0423** [comment]
@romer8 Done.

**Comment by romer8** [comment]
**1.	Requester Information:**

Giovanni Romero
email: gromero@aquaveo.com
slack: Gio

**2.	Project Information:** 
The following information will be needed to add the subdomain of the Tethys Platform portal.:

1. Recored Type: A
2. TTL: 1 Hour
3. Data  3.221.231.152 and  44.205.207.244

**3.	name of subdomain:** 
Request of the folllowing subdomain: [tethys.ciroh.org](http://tethys.ciroh.org/)

**4.	Url of serving address:**
The urls that the subdomain will be mapping to the following Network Load Balancer DNS:
cirohportal-prod-nlb-e606439f7706f620.elb.us-east-1.amazonaws.com

**Comment by benlee0423** [comment]
@romer8 
Remove existing one for [portal.ciroh.org](http://portal.ciroh.org/), replace with new one?

**Comment by romer8** [comment]
Yes, that is correct. I will still use [portal.ciroh.org](http://portal.ciroh.org/). I will submit a new PR for it

**Comment by benlee0423** [comment]
@romer8 
Done.
